pkg/controllers/dynakube/controller_test.go

```diff
@@ -455,7 +455,7 @@ func createVersionReconcilerBuilder(reconciler *mockversion.Reconciler) version.
 }
 
 func createInjectionReconcilerBuilder(reconciler *mockinjection.Reconciler) injection.ReconcilerBuilder {
-	return func(client client.Client, apiReader client.Reader, dynatraceClient dtclient.Client, istioClient *istio.Client, fs afero.Afero, dynakube *dynatracev1beta1.DynaKube) controllers.Reconciler {
+	return func(client client.Client, apiReader client.Reader, dynatraceClient dtclient.Client, istioClient *istio.Client, fs afero.Afero, dynakube *dynatracev1beta1.DynaKube, scheme *runtime.Scheme) controllers.Reconciler {
```

Comment:
```suggestion
	return func(_ client.Client, _ client.Reader, _*runtime.Scheme, _ dtclient.Client, _ *istio.Client, _ *dynatracev1beta1.DynaKube) controllers.Reconciler {
```

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/2777/commits/39d3778db07cc9d89ba637c88bb091975411c30b